app/routes/wallet.py

```
@@ -34,71 +31,42 @@ class DepositRequest(BaseModel):
-        wallet.balance += req.amount
+    wallet.balance += req.amount
     tx = Transaction(
         user_id=current_user.id,
         tx_type="deposit",
         amount=req.amount,
         currency=req.currency,
-        status=status,
-        risk_level=risk_level,
+        status="completed",
         reference=virtual_ref,
         description=f"Bank deposit via RTP"
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)